### ISSUE TITLE
malloc.h not found on Mac OS

### ISSUE THREAD

**Issue by stevenctl** [blaind/ttf2mesh-rs#10]
Seems like a similar issue was seen here: https://github.com/RIOT-OS/RIOT/pull/2590

I have xcode-tools (~mac's version of build-essential), clang and whatnot.

**Comment by stevenctl** [comment]
This issue probably makes more sense upstream.

**Comment by stevenctl** [comment]
https://github.com/fetisov/ttf2mesh/pull/8

**Comment by blaind** [comment]
Agreed, upstream fix would be preferred. Thank you for creating the issue.

If no upstream change is implemented, I the RIOT-OS approach can be followed in this wrapper as well. Probably can be incorporated into bindgen pipeline at https://github.com/blaind/ttf2mesh-rs/blob/main/ttf2mesh-sys/wrapper.h

**Comment by TethysSvensson** [comment]
Upstream has fixed this, I think you just need to bump the submodule.

**Comment by TethysSvensson** [comment]
Though upstream appears to still have a lot of other issues Mac OS

**Comment by J-Siu** [comment]
Any progress on this issue?
